app/components/windows/SourcesShowcase.vue

```diff
@@ -204,8 +223,14 @@
       v-if="inspectedSource === widgetTypes.DonationTicker"
       @clickAdd="selectWidget(widgetTypes.DonationTicker)"
       name="Donation Ticker"
-      description="Show off your most recent donations to your viewers.">
-      <img slot="media" src="../../../media/source-demos/donationticker.gif"/>
+      description="Show off your most recent donations to your viewers."
+      key="15">
+      <video class="source__demo source__demo--day" autoplay loop slot="media">
+        <source src="../../../media/source-demos/day/source-donation-ticker.mp4">
+      </video>
+      <video class="source__demo source__demo--night" autoplay loop slot="media">
```

Comment:
Does this work?  Rendering two elements into the same slot?

Reply:
Yep. One gets display: none based on the day/night mode

Reply:
Huh ok cool.